### PR TITLE
doc: add doc for url object

### PR DESCRIPTION
**Pull request by wong2**
fix: https://github.com/nodejs/undici/issues/240

### CODE REVIEW

**Review by ronag — CHANGES_REQUESTED** [review]
This has a lot of unrelated changes.

**Comment by wong2** [comment]
@ronag sorry for that, fixed now

File: README.md [hunk 1]
```diff
@@ -196,6 +196,16 @@ Gets the global dispatcher used by Common API Methods.
...
 
+### `UrlObject`
+
+* **port** `string | number | null` (optional)
+* **path** `string | null` (optional)
+* **pathname** `string | null` (optional)
+* **hostname** `string | null` (optional)
+* **origin** `string | null` (optional)
+* **protocol** `string | null` (optional)
+* **search** `string | null` (optional)
```
Comment (ronag) [inline README.md:207]: We don't support all of these properties. Please see lib/core/util/parseURL

Comment (wong2) [reply README.md:207]: I got the list from exactly [parseURL](https://github.com/nodejs/undici/blob/0b4694722d17c279e0e960b70046ea54654e254d/lib/core/util.js#L21). Which property should not be here in your opinion?

**Review by ronag — APPROVED** [review]

**Pull request merged**